docs/connector-development/connector-builder-ui/tutorial.md

```diff
@@ -0,0 +1,202 @@
+  "rates": {
+    "CAD": 1.28,
+    "EUR": 0.98
+  }
+}
+```
+
+## Setting up Exchange Rates API key
+
+Before we get started, you'll need to generate an API access key for the Exchange Rates API.
+This can be done by signing up for the Free tier plan on [Exchange Rates API](https://exchangeratesapi.io/):
+
+1. Visit https://exchangeratesapi.io and click "Get free API key" on the top right
+2. You'll be taken to https://apilayer.com -- finish the sign up process, signing up for the free tier
+3. Once you're signed in, visit https://apilayer.com/marketplace/exchangerates_data-api#documentation-tab and click "Live Demo"
+4. Inside that editor, you'll see an API key. This is your API key.
+
+## Setting up the environment
+
+Besides an Exchange Rates API key, only a browser and an up-to-date running Airbyte instance is required. There are two ways to accomplish this:
+* Coming soon: Sign up on [cloud.airbyte.com](https://cloud.airbyte.com/)
+* Download and run Airbyte [on your own infrastructure](https://github.com/airbytehq/airbyte#quick-start). Make sure you are runninging version 0.43.0 or later
+
+## Creating a connector builder project
+
+When developing a connector using the connector builder UI, the current state is saved in a connector builder project. These projects are saved as part of the Airbyte workspace and separate from your source configurations and connections. In the last step of this tutorial you will publish the connector builder project to make it ready to use in connections to run syncs.
+
+To get started, follow the following steps:
+* Visit the Airbyte UI in your browser
+* Go to the connector builder page by clicking the "Builder" item in the left hand navigation bar
+* Select "Start from scratch" to start a new connector builder project
+* Set the connector name to "Exchange rates"
+
+Your connector builder project is now set up. The next steps describe how to configure your connector to extract records from the Exchange Rates API.
+
+## Setting up global configuration
+
+On the "global configuration" page, general settings applying to all streams are configured - the base URL requests are sent to as well as configuration for how to authenticate with the API server.
+
+* Set the base URL to `https://api.apilayer.com`
+* Select the "API Key" authentication method
+* Set the "Header" to `apikey`
+
+The actual API Key you copied from apilayer.com will not be part of the connector itself - instead it will be set as part of the source configuration when configuring a connection based on your connector in a later step.
+
+## Setting up and testing a stream
+
+Now that you configured how to talk the API, let's set up the stream of records that will be sent to a destination later on. To do so, click the button with the plus icon next to the "Streams" section in the side menu and fill out the form:
+* Set the name to "Rates"
+* Set the "URL path" to `/exchangerates_data/latest`
+* Submit
+* Set the "Record selector" to `rates`
```

Comment:
I wanted to keep this out of the tutorial as it would require you to go into detail of what a primary key is doing. Let's leave that for the "Record selection" page